vnext/src/Libraries/Text/Text.windows.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import DeprecatedTextPropTypes from '../DeprecatedPropTypes/DeprecatedTextPropTypes';
+import StyleSheet from '../../StyleSheet/StyleSheet';
+import Text from '../../Text/Text.js';
+import TextAncestorBordered from './TextAncestorBordered.windows.js';
+import View from '../../Components/View/View';
+
+// [Windows]
+// Forking Text as a workaround in order to be able to render borders around it.
+function BorderedText(props) {
```

Comment:
I believe calling this function BorderedText might make all text components within the UI tree show up as BorderedText instead of Text in various debug tools and component stacks.  -- Try using flipper or something to look at your react tree and see what it looks like before/after this change.

Reply:
@acoates-ms Yes, it shows up in react devtools as BorderedText wrapping Text.
I've pushed an update rewriting this using code from upstream Text.js, rather than wrapping it.
There's a caveat, though: it only works fully in a standalone demo app. In RNTester, the path using the View tag as a wrapper doesn't update the context in the Provider tag, which results in nestedness detection being broken again.

If you think it's critical that we don't introduce a new component name in debugging tools, please help me address it. Otherwise, I'll revert the latest commit.